src/table/utils/handle-key-press.ts

```diff
@@ -124,7 +130,9 @@ export const handleTotalKeyDown = (
   rootElement: HTMLElement,
   cellCoord: [number, number],
   setFocusedCellCoord: React.Dispatch<React.SetStateAction<[number, number]>>,
-  isSelectionMode: boolean
+  isSelectionMode: boolean,
+  column: Column,
```

Comment:
We should try to reduce arguments here. So let's use the text on event.target instead. Same for header